backend/app/server/routes/user.py

```diff
@@ -1,41 +1,49 @@
-from fastapi import APIRouter, Body
+from fastapi import APIRouter, Body, status, Depends
 from fastapi.encoders import jsonable_encoder
-
+from fastapi.security import OAuth2PasswordBearer,OAuth2PasswordRequestForm
 from ..controllers.user import (
     add_user,
+    login,
     delete_user,
     retrieve_user,
-    update_user,
+    update_user,    
 )
+
+from ..controllers.auth import AuthHandler
 from ..models.user import (
     ResponseModel,
     UserSchema,
+    LoginSchema,
     UpdateUserModel,
 )
-
+auth_handler = AuthHandler()
```

Comment:
Should we not use the same AuthHandler object we created in Controllers? (there is one more in post router)